charts/atlantis/stages/production/values.yaml

```diff
@@ -0,0 +1,10 @@
+ingress:
+  enabled: false
+
+service:
+  type: LoadBalancer
```

Comment:
https://kubernetes.io/docs/concepts/services-networking/service/#aws-nlb-support

I was reading through the docs on load balancer source range and requires the use of an nlb when using the default externalTrafficPolicy of cluster.

In order to provision and nlb extra annotation is required.

```
  service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
```